electrochemistry.ttl

```diff
@@ -3303,25 +3694,44 @@ of other configurations are used."""@en ;
 
 ###  http://emmo.info/electrochemistry#electrochemistry_f0c24970_4c14_4207_bd78_5f2181a67085
 :electrochemistry_f0c24970_4c14_4207_bd78_5f2181a67085 rdf:type owl:Class ;
-                                                       rdfs:subClassOf :electrochemistry_f49b84d4_e1f9_424c_bb22_8cea23c0a7d4 ;
-                                                       emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "The process of forming a film or a bulk material using an electrochemical process where the electrons are supplied by an external power supply."@en ;
+                                                       rdfs:subClassOf :electrochemistry_2e3e14f9_4cb8_45b2_908e_47eec893dec8 ,
+                                                                       :electrochemistry_f49b84d4_e1f9_424c_bb22_8cea23c0a7d4 ;
+                                                       emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "the process of forming a film or a bulk material using an electrochemical process where the electrons are supplied by an external power supply"@en ;
                                                        skos:prefLabel "Electrodeposition"@en .
 
 [ rdf:type owl:Axiom ;
    owl:annotatedSource :electrochemistry_f0c24970_4c14_4207_bd78_5f2181a67085 ;
    owl:annotatedProperty emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 ;
-   owl:annotatedTarget "The process of forming a film or a bulk material using an electrochemical process where the electrons are supplied by an external power supply."@en ;
+   owl:annotatedTarget "the process of forming a film or a bulk material using an electrochemical process where the electrons are supplied by an external power supply"@en ;
    dcterms:source "A. J. Bard, G. Inzelt, and F. Scholz, Eds., Electrochemical Dictionary, 2nd Edition. Berlin: Springer-Verlag, 2012. DOI: https://doi.org/10.1007/978-3-642-29551-5"
  ] .
 
 
 ###  http://emmo.info/electrochemistry#electrochemistry_f1c7eacb_9f21_4100_925c_3974f266e06f
 :electrochemistry_f1c7eacb_9f21_4100_925c_3974f266e06f rdf:type owl:Class ;
-                                                       rdfs:subClassOf emmo:EMMO_a2a68924_b759_4ba4_a25d_126021569fb9 ;
-                                                       emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "a membrane composed of two layers, typically with different chemical properties or functional roles."@en ;
```

Comment:
I would keep the elucidation as is, is nice the context on how the two layers are different